src/Proto.Cluster/ClusterExtension.cs

```diff
@@ -30,6 +30,13 @@ public static Cluster Cluster(this IContext context)
         public static Task<T> ClusterRequestAsync<T>(this IContext context, string identity, string kind, object message, CancellationToken ct) =>
             //call cluster RequestAsync using actor context
             context.System.Cluster().RequestAsync<T>(identity, kind, message, context, ct);
+        
+        public static void ClusterRequestReenter<T>(this IContext context, string identity, string kind, object message, Func<Task<T>, Task> callback, CancellationToken ct)
```

Comment:
Use the ClusterIdentity overload?

Reply:
Then it could possibly have the same name as the other RequestReenter. In addition re-using ClusterIdentity is potentially more performant, as it can cache the activation pid internally.

Reply:
Added CI overload. but don't like the same name overload as we make the distinction everywhere else based on names. e.g. having RequestAsync and ClusterRequestAsync, but only RequestReenter seems inconsistent